server/common/dao/src/test/java/org/kaaproject/kaa/server/common/dao/AbstractTest.java

```diff
@@ -747,6 +743,7 @@ protected EndpointProfileDto generateEndpointProfileDto(String appId, List<Strin
         profileDto.setSubscriptions(topicIds);
         profileDto.setEndpointKeyHash("TEST_KEY_HASH".getBytes());
         profileDto.setServerProfileBody("{\"serverTitle\": \"SERVER_TEST\"}");
+        profileDto.setEpsConfigurationHash(EPS_CONFIG_HASH);
```

Comment:
Verified v0.10 tests with old cassandra db schema. Change is not backward compatible.